sdk/src/main/java/io/radar/sdk/Radar.kt

```diff
@@ -974,6 +979,7 @@ object Radar {
         near: Location,
         radius: Int,
         tags: Array<String>?,
+        metadata: Map<String, String>?,
```

Comment:
super small point - it looks here that if `metadata` is present, `tags` must be present too. Are we allowing customers to search on metadata without a tag?

Reply:
I guess they'd explicitly pass `nil` or `[]` to get around this

Reply:
Ah yeah, they might have to pass in `[]` for tags.  Would be nice to not require that, but I'm not sure there's a cleaner way to do this.  Probably just one of those limitations that goes along with multiple optional params